### PR TITLE
Creating guvs

### PR DESCRIPTION
**Pull request by GingertronMk1**
fuck it's got a lot of people in it

Also wait until final show, add prod shots, etc, before merging

### CODE REVIEW

File: _shows/17_18/one_man_two_guvnors.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  name: Miguel Barrulas
+- role: Shadow Sound
+  name: Jakub Kolek
+- role: Improv Consultant
+  name: Jack Ellis
+- role: Technical Operator
+  name: Nathan Penney
+- role: Technical Operator
+  name: Darcey Graham
+- role: Technical Operator
```
Comment (samozzy) [inline _shows/17_18/one_man_two_guvnors.md]: Replace Darcey with Zoe Smith; Darcey is now a stage manager.

Comment (GingertronMk1) [reply _shows/17_18/one_man_two_guvnors.md]: Done

**Comment by GingertronMk1** [comment]
Also I'll have one last commit to make after the final show, don't merge til that happens

**Comment by GingertronMk1** [comment]
Unless someone finds something massively wrong this is good to go

**Pull request merged**